packages/lime-proto-bmx6/src/bmx6.lua

```diff
@@ -31,6 +31,17 @@ function bmx6.configure(args)
 	uci:set(bmx6.f, "main", "tun4Address", ipv4:string())
 	uci:set(bmx6.f, "main", "tun6Address", ipv6:string())
 
+	-- If publish own IP enabled, configure tunIn
+	local pub_own_ip = config.get_bool("network", "bmx6_publish_ownip", false)
+	if (pub_own_ip) then
+		uci:set(bmx6.f, "myIP4", "tunIn")
+		uci:set(bmx6.f, "myIP4", "tunIn", "myIP4")
+		uci:set(bmx6.f, "myIP4", "network", utils.split(ipv4:string(),'/')[1]..'/32')
+		uci:set(bmx6.f, "myIP6", "tunIn")
+		uci:set(bmx6.f, "myIP6", "tunIn", "myIP6")
+		uci:set(bmx6.f, "myIP6", "network", utils.split(ipv6:string(),'/')[1]..'/128')
```

Comment:
please use library function to get IP instead of utils.split as show here
https://github.com/libremesh/lime-packages/pull/278/files#diff-de438b5b838191b385771cf207bc581cR41